### ISSUE TITLE
Package problem?

### ISSUE THREAD

**Issue by oferRounds** [MacKentoch/react-native-beacons-manager#48]
Hi,

Thanks for this great repo!

I run `npm install react-native-beacons-manager` (+ linking), but for some reason, the `react-native-beacons-manager.d` file lacks the **iOS only** part (these line are missing), i.e., the file is different than master.

Any idea what might cause this? I do see that I have the latest version installed (1.0.7).

**Comment by MacKentoch** [comment]
Hi @oferRounds 

It may be related to a breaking change fixed in master (v1.0.8).

Still an amount of work needed before publishing `v1.0.8` to npm so as a workaround you could use master version:

```bash
yarn add git+https://github.com/MacKentoch/react-native-beacons-manager.git
```

**Comment by oferRounds** [comment]
@MacKentoch thanks!
